spec/controllers/groups/memberships_controller_spec.rb

```diff
@@ -0,0 +1,3 @@
+RSpec.describe Groups::MembershipsController, type: :controller do
+
```

Comment:
Can you add test coverage for this?

Reply:
Need a pair for helping me in writing tests because as of now, I don't know how to write tests yet. :sweat_smile: